### DISCUSSION TITLE
gencode release version

### DISCUSSION THREAD

**Discussion by SimonLi5601** [broadinstitute/seqr#1947] [Q&A]
I was wondering why the gencode version only up to release 31? It seems it has the release of 38 on May 2021. Thanks!
https://www.gencodegenes.org/human/releases.html

**Comment by hanars** [comment]
Are you asking about in seqr itself, or in the loading pipleine? In seqr, we try to keep the gencode version in line with the latest version the pipeline is running, otherwise we start to see really confusing gene annotations on variants. For the pipeline, I think we just haven't had the bandwidth to update it

**Comment by SimonLi5601** [comment]
I was asking the seqr itself, but you did answer my question. Thank you!
